autoreject/autoreject.py

```diff
@@ -300,7 +300,7 @@ def _compute_thresh(this_data, method='bayesian_optimization',
         cache = dict()
 
         def func(thresh):
-            idx = np.argmin(np.abs(thresh - all_threshes))
+            idx = np.where(thresh - all_threshes >= 0)[0][0]
```

Comment:
Thresholds are too low as you can see ... ![bad_segments](https://user-images.githubusercontent.com/15852194/28503639-f3031e64-700a-11e7-8295-f967a643a57a.png)